roles/kafka_connect/tasks/deploy_connectors.yml

```diff
@@ -4,17 +4,17 @@
   with_items: "{{groups['kafka_connect']}}"
 
 - name: Register connector configs and remove deleted connectors for single cluster
-  kafka_connectors:
-    connect_url: "{{kafka_connect_http_protocol}}://{{ hostvars[inventory_hostname]|resolve_hostname }}:{{kafka_connect_rest_port}}/connectors"
+  confluent.platform.kafka_connectors:
```

Comment:
Nice catch! Did the jenkins run test this module?

I'm wondering if we want to standardize on collections naming scheme on other modules eventually.

Reply:
Yes, Jenkins runs this module. That is how I caught this. 👍